platforms/software/testing-base/src/main/resources/org/gradle/api/internal/tasks/testing/report/generic/style.css

```diff
@@ -11,6 +12,14 @@
     vertical-align: top;
 }
 
+div.tab table.thin-cols td {
+    padding-left: 0.5em;
+}
+
+div.tab table.thin-cols th {
+    padding-left: 0.5em;
+}
```

Comment:
💅 Instead of `thin-cols` (which only describes the intended format), it would be better to add a class that describes the purpose of the element, and then apply the style to it.  Maybe `test-results`?